electricwatch.py

```diff
@@ -0,0 +1,7 @@
+N=input('N = ')
+N=int(N)
+hours = N // 3600
+N = N - hours*3600
```

Comment:
Запись можно упростить:
```suggestion
N -= hours*3600
```